data/conversations.txt

```diff
@@ -10,31 +10,33 @@
 
 conversation "intro"
 	scene "scene/lobby"
-	`The bank's architecture is what you would have called "futuristic" back when you dreamed that the future would be less squalid than the present: story above story of curves and spires and balconies, all in gleaming metal. A doorman stands by each of the heavy glass doors. You are wearing your very best clothes, but you feel shabby next to them.`
-	`	The loan broker's office is equally intimidating. Large computer monitors fill the walls, with stock quotes and other information spilling down them almost too fast to read. You occasionally glimpse the names of places that you have only seen in movies: Aldebaran. Tarazed. Earth.`
-	`	The pen is heavy as lead but glides almost without friction as you print your initials on the bottom of page after page of ominous legal documents and sign several dozen statements, notices, and agreements. The banker, a balding middle-aged man in a suit that you suspect costs more than the spaceship you are about to purchase, flips the pages past you quickly, uttering a curt, "Sign here. Now here," as he points to each page. He moves swiftly, mechanically.`
+	`The bank's architecture is what you would have called "futuristic" back when you dreamed that the future would be less squalid than the present: story above story of curves and spires and balconies, all in gleaming metal. A doorman stands by each of the heavy glass doors. Despite wearing your very best outfit you still feel inadequate next to their glittering uniforms.`
+	`	The loan broker's office is equally intimidating. Impressive computer monitors fill the walls, with stock quotes and finance data spilling down them almost too fast to read. You occasionally glimpse the names of destinations that you have only seen in movies: Aldebaran. Tarazed. Earth.`
+	`	The pen is heavy as lead but glides almost without friction as you print your initials on the bottom of page after page of ominous legal documents and sign several dozen statements, notices, and agreements. The banker, a balding middle-aged man in a suit that you suspect costs more than the spaceship you are about to purchase, nimbly flips through this dense river of legalese, uttering a curt, "Sign here. Now here," as he points to each page. He moves swiftly, mechanically.`
```

Comment:
Too informal for the game.

Reply:
is it the inclusion of the word legalese?

Reply:
`this dense river of legalese`